Makefile

```diff
@@ -1,56 +1,56 @@
-default: prepare-build
-default: build/holo build/holo-files
-default: build/man/holorc.5 build/man/holo-plugin-interface.7 build/man/holo-test.7 build/man/holo.8 build/man/holo-files.8
+BUILD_TARGETS += $(patsubst cmd/%,.gopath/bin/%, $(wildcard cmd/*))         # build all binaries
+BUILD_TARGETS += $(patsubst doc/%.pod,.gopath/man/%, $(wildcard doc/*.pod)) # build all manpages
+
+default: $(BUILD_TARGETS)
 
 VERSION := $(shell ./util/find_version.sh)
 
+GO            := env GOPATH=$(CURDIR)/.gopath GOBIN=$(CURDIR)/.gopath/bin go
 GO_BUILDFLAGS :=
 GO_LDFLAGS    := -s -w
 
-prepare-build:
-	@mkdir -p build/man
-build/holo: src/holo/main.go src/holo/*/*.go
-	go build $(GO_BUILDFLAGS) --ldflags "$(GO_LDFLAGS) -X main.version=$(VERSION)" -o $@ $<
-build/holo-files: src/holo-files/main.go src/holo-files/*/*.go
-	go build $(GO_BUILDFLAGS) --ldflags "$(GO_LDFLAGS)" -o $@ $<
+.gopath/bin/holo: cmd/holo/main.go lib/holo/*.go
+	$(GO) install $(GO_BUILDFLAGS) --ldflags "$(GO_LDFLAGS) -X github.com/holocm/holo/cmd/holo.version=$(VERSION)" github.com/holocm/holo/cmd/holo
```

Comment:
This is wrong; the `main` package is special to the linker; it still needs to be `-X main.version=$(VERSION)`.